analysis_module/models.py

```diff
@@ -0,0 +1,23 @@
+import uuid
+from django.db import models
+
+
+class AnalysisModuleStatus(models.Model):
+
+    class RequestStatus(models.IntegerChoices):
+        INITIATED = 1
+        SUCCESS = 2
+        FAILED = 3
+
+    class FeaturesType(models.TextChoices):
+        NGRAMS = "ngrams", "Ngrams"
+        TOPICMODEL = "topicmodel", "Topicmodel"
+        SUMMARIZATION = "summarization", "Summarization" 
+
+    status = models.IntegerField(choices=RequestStatus.choices, default=RequestStatus.INITIATED)
+    unique_id = models.UUIDField(default=uuid.uuid4, editable=False)
+    result_s3_link = models.URLField()
+    type = models.CharField(choices=FeaturesType.choices, max_length=20)
+
+    class Meta:
```

Comment:
We don't need explicit db table name, the table is automatically created and managed by `makemigrations` and `migrate` commands.

Reply:
all the db updates are made not by django but in the ranjan handlers inside the spinned container, i think that he need a specific table name, because the django model is not present there, right? maybe i'm missing something

Reply:
example: https://github.com/the-deep-nlp/core-server-infra/blob/feature/modules/handlers/ecs/summarization/app.py